lib/ui/painting.dart

```diff
@@ -1830,6 +1835,49 @@ class Image {
   String toString() => _image.toString();
 }
 
+class ImagePerfData {
+  ImagePerfData(this.resourceDecodeTime);
+  int resourceDecodeTime;
+}
+
+// ignore: avoid_classes_with_only_static_members
+class ImagePerfInfos {
```

Comment:
if @chinmaygarde recommends we continue using extension methods, these statics could all be merged in with the extension method class which would comply with the lint. Also consider making the names all private if they aren't used outside of this class. It could be useful to see the corresponding change to package:flutter and package:svg for some more clarity on how this is used.